File: Q.14 CheckPrime((coding NInjas)/CheckPrime.py
from math import *
from collections import *
from sys import *
from os import *

## Read input as specified in the question.
## Print output as specified in the question.
n=int(input())
count=0
for i in range(2,int(sqrt(n))+1):
    if(n%i==0):
        count+=1
if(count>0 or n==1):
    print("NO")
else:
    print("YES")

File: Q.14 CheckPrime((coding NInjas)/CheckPrime.java
import java.util.Scanner;
public class CheckPrime {
    public static String isPrime(int num) {
		//Your code goes here
		int count=0;
		for(int i=2;i<=Math.sqrt(num);i++){
            if(num%i==0){
                count+=1;
            }
		}
		
		return (count>0 || num==1)?"NO":"YES";
	}

    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int n = sc.nextInt();
        System.out.println(isPrime(n));
        sc.close();
    }
    
}
